### PR TITLE
[SALEOR-6519] app-bridge-based JWT token and domain used to fetch data from GraphQL

### PR DESCRIPTION
**Pull request by magul**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +0,0 @@
-NEXT_PUBLIC_SALEOR_API_URL=https://vercel.saleor.cloud/graphql/
```
Comment (magul) [inline .env:1]: should I remove that whole file?

File: providers/GraphQLProvider.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                ...fetchOptions,
+                headers: {
+                  ...fetchOptions.headers,
+                  "Authorization-Bearer": authState.token,
+                },
+              },
+            },
+          };
+        },
+        getAuth: async ({ authState, mutate }) =>  {
```
Comment (magul) [inline providers/GraphQLProvider.tsx:43]: @zaiste jwm0 this is a place, that we need to figure out how app-bridge based JWT tokens are rotated.

**Review by Cloud11PL — APPROVED** [review]

File: providers/GraphQLProvider.tsx (hunk 2 above)
Comment (jwm0) [reply providers/GraphQLProvider.tsx:43]: If we just call `getState()` here we'll always get fresh token

Comment (magul) [reply providers/GraphQLProvider.tsx:44]: I've just checked (using `setInterval`) and on the same app (meaning I cannot run `useApp` outside of function component), I tried to
```
console.log(app.getState().token)
```
and it was always the same and eventually expired

Comment (magul) [reply providers/GraphQLProvider.tsx:44]: I've briefly checked dashboard and app-bridge and haven't found any means to refresh this token, when needed.

**Comment by magul** [comment]
Based on @Cloud11PL approve I'm merging that one and we will handle refreshing token in subsequent pull request (after we figure out how to do that with @jwm0 ).

**Pull request merged**